Pollo/ViewControllers/PollsDateViewController+Extension.swift

```diff
@@ -120,22 +120,8 @@ extension PollsDateViewController: PollBuilderViewControllerDelegate {
 extension PollsDateViewController: NavigationTitleViewDelegate {
 
     func navigationTitleViewNavigationButtonTapped() {
-        guard userRole == .admin else { return }
```

Comment:
would be good to remember we have this stored somewhere so we don't have to rewrite it

Reply:
i don't think we ever will rewrite it because it's all like past session fetching which we won't even have to do for the future settings page